Gruntfile.js

```diff
@@ -0,0 +1,231 @@
+const path = require('path')
```

Comment:
It can be nice to split a Gruntfile up into separate files kept in a `./grunt/` directory. Gruntfiles often get quite big an unwieldy.